tests/StatsdClient.Tests/CommandIntegrationTests.cs

```diff
@@ -46,51 +45,30 @@ public void ClearUdpListenerMessages()
             _udpListener.GetAndClearLastMessages(); // just to be sure that nothing is left over
         }
 
-        // Test helper. Waits until the listener is done receiving a message,
-        // then asserts that the passed string is equal to the message received.
-        private void AssertWasReceived(string shouldBe, int index = 0)
-        {
-            // Stall until the the listener receives a message or times out
-            while (_listenThread.IsAlive) ;
-            Assert.AreEqual(shouldBe, _udpListener.GetAndClearLastMessages()[index]);
-        }
-
-        // Test helper. Waits until the listener is done receiving a message,
-        // then asserts that the passed regular expression matches the received message.
-        private void AssertWasReceivedMatches(string pattern, int index = 0)
-        {
-            // Stall until the the listener receives a message or times out
-            while (_listenThread.IsAlive) ;
-            StringAssert.IsMatch(pattern, _udpListener.GetAndClearLastMessages()[index]);
-
-        }
-
         [Test]
-        public void _udp_listener_sanity_test()
+        public void Udp_listener_sanity_test()
```

Comment:
ditto about CamelCase vs snake_case. I'm not sure what the convention is, just want to make sure if we stick to snake_case it's for good reason.